src/invidious/views/components/player.ecr

```diff
@@ -4,7 +4,7 @@
     <% if params.autoplay %>autoplay<% end %>
     <% if params.video_loop %>loop<% end %>
     <% if params.controls %>controls<% end %>>
-    <% if (hlsvp = video.hls_manifest_url) && !CONFIG.disabled?("livestreams") %>
+    <% if (hlsvp = video.hls_manifest_url) && hlsvp.includes?("/source/yt_live_broadcast/") && !CONFIG.disabled?("livestreams") %>
```

Comment:
You should just be able to check whether `video.live_now` is true instead of the URL

Reply:
Thanks - great catch. I'm still digging through the codebase, this does a better job that checking the URL. 

I've updated to that in case it's worth merging this in in the meantime as some videos are broken at the moment.